client/blocks/reader-join-conversation/dialog.jsx

```diff
@@ -0,0 +1,49 @@
+import { Dialog } from '@automattic/components';
+import { Button } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import { useState } from 'react';
+import LoginBlock from 'calypso/blocks/login';
+//import WPLogin from 'calypso/login/wp-login';
+
+import './style.scss';
+
+const ReaderJoinConversationDialog = ( { onClose, isVisible } ) => {
+	const translate = useTranslate();
+	const [ isLoginVisible, setIsLoginVisible ] = useState( false );
+	const showLogin = () => {
+		setIsLoginVisible( true );
+	};
+	return (
+		<Dialog
+			additionalClassNames="reader-join-conversation-dialog"
+			isBackdropVisible={ true }
+			isVisible={ isVisible }
+			onClose={ onClose }
+			showCloseIcon={ true }
+			label={ translate( 'Join the conversation' ) }
+			shouldCloseOnEsc={ true }
+		>
+			{ ! isLoginVisible && (
+				<div className="reader-join-conversation-dialog__content">
+					<h1>{ translate( 'Join the conversation' ) }</h1>
+					<p>
+						{ translate( 'Sign in to like, comment, reblog, and follow your favorite blogs.' ) }
+					</p>
+					<Button
+						isTertiary
+						onClick={ showLogin }
```

Comment:
I think this should open a new tab to create an account or something? Or would we build a signup form in here as well?

Reply:
This will (eventually) open the login popup on the account creation page (if that's possible)